pkg/kubelet/lifecycle/handlers.go

```diff
@@ -149,6 +152,10 @@ func (hr *handlerRunner) runHTTPHandler(pod *v1.Pod, container *v1.Container, ha
 
 			// clear err since the fallback succeeded
 			if httpErr == nil {
+				if eventRecorder != nil {
```

Comment:
Delete todo at line 144
```
// TODO: emit an event about the fallback
```

Can you move the metric TODO down to here while you're at it?

Reply:
done